### ISSUE TITLE
  Fatal error: Class 'SimpleSAML_Auth_Simple' not found

### ISSUE THREAD

**Issue by jhayiwg** [ragingdave/simplersaml#4]
I`m trying to use your library unfortunately I`m getting this issue.

[Symfony\Component\Debug\Exception\FatalThrowableError]  
  Fatal error: Class 'SimpleSAML_Auth_Simple' not found

Many Thanks

**Comment by ragingdave** [comment]
Sorry I hadn't responded sooner, but this library requires that you have the simplesamlphp library installed.

**Comment by soltmar** [comment]
Why not to add entry in "require" in composer.json of your library if it actually requires it :)

**Comment by ragingdave** [comment]
Good Point! I will try to fix this and the other issues once I have a few minutes to set up a test environment.
